screensaver/screensaver.py

```diff
@@ -74,86 +93,269 @@ def get_last_level_directories(photo_dirs_list) -> list:
             if line in line2:
                 counter += 1
         if counter == 1:
-            photo_dirs_list_final.append(line.rstrip())
+            final_dir = os.path.join(line.rstrip(), '')
+            photo_dirs_list_final.append(final_dir)
     return photo_dirs_list_final
 
 
-def delete_old_screensaver_photos() -> None:
+def rsync_directory(source, destination) -> None:
+
```

Comment:
Please add a docstring to summarise what this function does and why it is different from other use cases of subprocess in the code, e.g. in `get_remote_dirs_list`, etc.